src/modules/configuration/index.ts

```diff
@@ -316,6 +316,15 @@ export const configValues = () => ({
         address: process.env.OP_REWARDS_MERKLE_DISTRIBUTOR_ADDRESS,
         blockNumber: Number(process.env.OP_REWARDS_MERKLE_DISTRIBUTOR_BLOCK_NUMBER),
       },
+      arbRewards: {
+        chainId: process.env.ARB_REWARDS_MERKLE_DISTRIBUTOR_CHAIN_ID
+          ? Number(process.env.ARB_REWARDS_MERKLE_DISTRIBUTOR_CHAIN_ID)
+          : 42161,
+        address: process.env.ARB_REWARDS_MERKLE_DISTRIBUTOR_ADDRESS || "0x9f6Cb0A37F1ae91b8e65405f525A596bAFC5A9a6",
+        blockNumber: process.env.ARB_REWARDS_MERKLE_DISTRIBUTOR_BLOCK_NUMBER
+          ? Number(process.env.OP_REWARDS_MERKLE_DISTRIBUTOR_BLOCK_NUMBER)
+          : 220861533,
+      },
```

Comment:
The config values for the MerkleDistributor contract used for distributing the referral rewards. The fallback values represent the contract deployed on mainnet